src/Console/XdebugHandler.php

```diff
@@ -52,7 +52,7 @@ public static function check(LoggerInterface $logger): void
     {
         // We force the color option unconditionally since it is able to detect the --no-ansi option
         // to disable it if necessary
```

Comment:
is this comment still relevant since no `--ansi` argument is passed anymore?

Reply:
removed